modules/channel-web/src/views/lite/components/Composer.tsx

```diff
@@ -101,7 +108,7 @@ class Composer extends React.Component<ComposerProps, { isRecording: boolean }>
       )
 
     return (
-      <div role="region" className={'bpw-composer'}>
+      <div role="region" className={classNames('bpw-composer', direction)}>
```

Comment:
Does this still work when direction is null?

Reply:
When `direction` is undefined it doesn't add a class, which is helpful for keeping the inherited direction. Checkout [this gif](https://github.com/botpress/botpress/pull/5692#discussion_r754314204)